### ISSUE TITLE
Wifi off/on freeze the settings page

### ISSUE THREAD

**Issue by startcloud** [ubports/ubuntu-touch#316]
- Device: Nexus 5
- Channel: stable
- Build: 15.04 r2 

## Steps to reproduce

1. Phone connected to wifi 
2. Phone is disconnected 
3. phone is not able to see the SSID 
4. try to turn off and on the wifi

## Expected behavoir
Phone should work fine and wifi is connected to the requested wireless network 

## Actual behavior
wifi stays off and settings page is blocked and phone is not working fine 
A reboot is needed to back to a stable state 

## Logfiles and additional information
I don't know which kind of logs you want to add to this issue

**Comment by NeoTheThird** [comment]
Not reproducible here, sorry. Could you provide logs as described on https://docs.ubports.com/en/latest/contribute/bugreporting.html#getting-logs ? Thanks :)

**Comment by startcloud** [comment]
I am able to find a scenario to reproduce 
- Reboot the phone 
- The phone is not connected to internet ( using wifi only ) (I don't know why the phone is not associated automatically) 
- Start an application that tries to make network traffic  
- Try to enable/disable the wifi to connect 
=> get the issue
[UTdmesg.txt](https://github.com/ubports/ubuntu-touch/files/1424421/UTdmesg.txt)
[UTlogcat.txt](https://github.com/ubports/ubuntu-touch/files/1424422/UTlogcat.txt)

**Comment by simoxxx** [comment]
happened to me too .. on r278

**Comment by Valkhir** [comment]
I think I am seeing this same issue on `bacon` running OTA-3. Did not see the issue before updating. Only other recent change: started using, intermittently, a bluetooth keyboard with the phone. I am not sure if the issue has happened when bluetooth was off. On the other hand, it has definitely happened when bluetooth was on.
Usually, I think the switch in the notification shade should execute `rfkill block wifi / rfkill unblock wifi` and then change UI state, but when this issue is happening, only UI state changes, the underlying command is not executed or has no effect (`rfkill list` shows no change, should show state change in `soft-blocked` for wifi). 

Executing `rfkill` manually from command line works, state change reflects both in notification shade GUI and `rfkill list`. This is a workaround if you don't want to reboot the phone.

Question to the system gurus here: is there a service or combination of services we can restart to fix this disconnect between UI state and `rfkill` state? Because restarting the phone is tedious and manually `rfkill`ing does not update the UI and only works around the issue...

**Comment by Arc676** [comment]
In case it's relevant, I'd like to add that toggling the WiFi from settings on the `devel` version (r351) has no negative side effects; it all works fine.

Supposedly this means that whatever causes the freezing was fixed between the stable version and the current development build?

**Comment by Flohack74** [comment]
Can be. As we did not work much on 15.04 anymore, you are advised to consider 15.04-devel as the latest stable. Do not install stable 15.04 anymore.

**Comment by Flohack74** [comment]
Yes, 15-04 devel ws not updated since december and probably will stay like this.

**Comment by NeoTheThird** [comment]
Not reproducible in 16.04/devel. @startcloud could you confirm that? If it's no longer an issue, we can add it to the milestone.

**Comment by fw190d13** [comment]
This is a constatnt error for me. I do a fresh install of 16.04 and it works OK., Than there is a OTA which I aply and after reboot there is no Wi-Fi at all so I have to do a fresh install again via the laptop to have a working phone. This bug is there for the last 2-3 months I think.

**Comment by WillemHexspoor** [comment]
I can't reproduce on a Nexus 5 running 16.04/stable ota-6 what startcloud describes, also after getting OTA-6 there was no issue that resembles what fw190d13 describes.

**Comment by UniversalSuperBox** [comment]
Closing from "works for me" comment.

If you are still experiencing this issue, please open a new issue with the updated relevant logs.
